src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -3,6 +3,7 @@
   name: 'simple_instance',
   root: true,
   private: true,
+  crossTenantQueriesEnabled: true,
```

Comment:
I should move this to the composite instance ET. Otherwise, it will get inherited by composite_items and anything else that uses it, which could be confusing